ci/simple-tests/coq-test-coqtop-unavailable.el

```diff
@@ -18,7 +18,7 @@
   (setq coq-prog-name "unavailable-program")
 
   ;; ensure coq-prog-name cannot be found
-  (should (not (locate-file coq-prog-name exec-path)))
```

Comment:
Could you explain this change?

Reply:
As I mentioned earlier, I wanted to appropriate this project by making sure my building was fine, so I broke a simple test first. But now I think my PR should be a bit cleaner.